ee/clickhouse/queries/funnels/base.py

```diff
@@ -119,9 +124,39 @@ def _format_single_funnel(self, results, with_breakdown=False):
             else:
                 serialized_result.update({"average_conversion_time": None, "median_conversion_time": None})
 
+            # Construct converted and dropped people urls. Previously this logic was
+            # part of
+            # https://github.com/PostHog/posthog/blob/e8d7b2fe6047f5b31f704572cd3bebadddf50e0f/frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepTable.tsx#L483:L483
+            # NOTE: we default to 0 on order being None. On the frontend the typing
```

Comment:
Since we're here, let's use `step.index` ! `order` is deprecated, and `index` is the new prop guaranteed to be there, in order.

https://github.com/PostHog/posthog/blob/master/posthog/models/filters/mixins/common.py#L345